src/runtime/recipe/handle-connection.ts

```diff
@@ -146,6 +146,16 @@ export class HandleConnection implements Comparable<HandleConnection> {
     const slandle: SlotConnection = new SlotConnection(this.name, this.particle);
     slandle.tags = this.tags;
     slandle.targetSlot = this.handle && this.handle.toSlot();
+    if (this.spec) {
+      this.spec.dependentConnections.forEach(connSpec => {
+        const conn = this.particle.getConnectionByName(connSpec.name);
+        if (!conn) return;
```

Comment:
AFAIK, we generally require braces for all our if-statements:

```
if (!conn) {
  return;
}
```

If I'm wrong and this is acceptable team style, then just ignore me.

Reply:
The linter is set to ignore them if they're on the same line. It might be good to be more strict but I think the 'same line' requirement catches the cases where I'd be concerned about it (e.g. Heartbleed)